companion.py

```diff
@@ -47,7 +52,7 @@
     'Narcotics'     : 'Legal Drugs',
     'Slaves'        : 'Slavery',
     'Waste '        : 'Waste',
-    'NonMarketable' : False,	# Don't appear in the in-game market so don't report
+    'NonMarketable' : False  # Don't appear in the in-game market so don't report
```

Comment:
I prefer the trailing comma stay, makes adding a next line less work.  A diff would show only the new line, not the addition of the command to the prior line as well.